README.md

```diff
@@ -91,17 +91,17 @@ module.exports = defineConfig({
 npx cypress run
 ```
 
-2. If there are failed tests, run the following command from the **directory of the project's `cypress.config`**:
+2. If there are failed tests, run the following command from the **failedTestDirectory of the project**:
```

Comment:
The `failedTestDirectory` change here assumes the user will be customizing the path for storing failed tests. I'd rather keep it as an option within the plugin to set the environment variable `failedTestDirectory`.